src/web/src/app/space/invitation-list/component.ts

```diff
@@ -215,9 +214,12 @@ export class SpaceInvitationListComponent implements OnInit {
 	}
 
 	approveInvitation(item: Detail) {
+		if (!item.role){
```

Comment:
Добавь пробел пожалуйста везде между круглой и фигурной скобкой
```suggestion
		if (!item.role) {
```

Reply:
Готово (везде).